cmd/sheltertech-go/main.go

```diff
@@ -74,12 +76,20 @@ func main() {
 	r.Use(prometheusMiddleware)
 	r.Use(middleware.Logger)
 	r.Use(sentryHandler.Handle)
+
 	r.Get("/api/categories", categoriesManager.Get)
 	r.Get("/api/categories/{id}", categoriesManager.GetByID)
 	r.Get("/api/categories/subcategories/{id}", categoriesManager.GetSubCategoriesByID)
 	r.Get("/api/categories/featured", categoriesManager.GetByFeatured)
 	r.Get("/api/categories/counts", categoriesManager.GetCategoryCounts)
 
+	// no user auth in folders yet
+	r.Get("/api/folders/user/{user_id}", foldersManager.Get)
```

Comment:
For this, I was thinking that the user ID would be a query parameter, so that it would look like this:

```
/api/folders?user_id=1
```

This would give us a bit more flexibility in the future in terms of adding filter parameters or even allowing us to filter on a list of user IDs. It would also have the benefit of making this look more like a normal (or at least Rails-ish) RESTful route.

Reply:
I made this change in https://github.com/ShelterTechSF/sheltertech-go/pull/115/commits/6b9dfea690bb046a4c376b06e1e7897d619add71